packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -119,6 +119,10 @@ chmod +x %{buildroot}%{gem_instdir}/extra/dynflow-debug.sh
 %{__mkdir_p} %{buildroot}%{foreman_dir}/script/foreman-debug.d
 ln -s %{gem_instdir}/extra/dynflow-debug.sh %{buildroot}%{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
 
+# Link rake task interface scripts into /usr/bin
+ln -s %{gem_instdir}/extra/foreman-tasks-cleanup.sh %{buildroot}%{_bindir}/foreman-tasks-cleanup.sh
```

Comment:
`ln: failed to create symbolic link '/builddir/build/BUILDROOT/rubygem-foreman-tasks-5.1.0-1.fm3_1.el8.noarch/usr/bin/foreman-tasks-cleanup.sh': No such file or directory`